docs/QuadraticSolver.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+activate Command
+Command -> Quadratic: quadratic.solveQuadratic()
+
+activate Quadratic
+Quadratic -> Quadratic: findA()
+activate Quadratic #FFBBBB
+deactivate Quadratic
+Quadratic -> Quadratic: findB()
+activate Quadratic #FFBBBB
+deactivate Quadratic
+Quadratic -> Quadratic: findC()
+activate Quadratic #FFBBBB
+deactivate Quadratic
+Quadratic -> Quadratic: quadraticFormula()
+activate Quadratic #FFBBBB
+deactivate Quadratic
+Quadratic -> Quadratic: printAnswer()
+activate Quadratic #FFBBBB
+Quadratic -> UI: Ui.printQuadraticAnswer()
+activate UI
+deactivate UI
+deactivate Quadratic
+opt Exceptions
+Quadratic -> UI: printQuadraticFormatError()
+activate UI
+deactivate UI
+end opt
+deactivate Quadratic
+Quadratic -> Command
```

Comment:
I believe in this case, the arrow should be a dashed arrow since it represents a method return. Also, you should try to connect the method return arrow to exactly the end of the activation bar for `Quadratic`